shunting-yard-binary/src/main/resources/shunting-yard-minimal.yml.template

```diff
@@ -13,3 +13,10 @@ source-table-filter:
   table-names:
     - db1.table_1
     - db2.table_2
+table-replications:
```

Comment:
Not sure if this is used in a test, but you probably still want the default where there are no table-replications in the template. I still think no override will be the most common use case.

Reply:
Its not used in a test. Just a reference for all supported config options. I need to write a proper README for Shunting Yard similar to what we have for CT. For now, It acts as a good spot from which you can copy-paste config.